### PR TITLE
Created mobile menu

### PR DESCRIPTION
**Pull request by 10-menachi**
Added:
- Mobile Menu
- Interactive Close Button on mobile menu
- Interactive ham menu

### CODE REVIEW

**Review by aamirkhan2478 — COMMENTED** [review]
Hi @10-menachi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: :red_circle: 

You have submitted a project, that uses the wrong GitHub flow check the below :point_down:  links for more details  :+1: 

1. How to use GitHub flow [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)
1. How to ask for a code review at Microverse [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/how_to_ask_for_a_code_review.md)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -438,9 +443,43 @@ textarea::placeholder {
...
+  }
+
+  .mobile-menu ul li {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1.5rem;
+    line-height: 2.25rem;
+  }
+
```
Comment (pinkmoon25) [inline styles.css]: - [ ] As one of the goals of the project is to parse a figma design as closely as possible, I think we can improve the navigation links here, according to the figma design the **font-size should be 32px(2rem)** currently they are 24px, also the **font-weight should be 600** currently they are set to 500. please fix this :+1:

File: index.html [hunk 2]
```diff
@@ -14,25 +14,43 @@
...
-        <a href="#portfolio" class="logo-text">Chris</a>
+        <a href="#" class="logo-text">Chris</a>
       </div>
 
       <div class="nav-links">
         <ul>
-          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#">Portfolio</a></li>
           <li><a href="#about">About</a></li>
           <li><a href="#contact">Contact</a></li>
```
Comment (pinkmoon25) [inline index.html]: - [ ] Please make sure the navigation links when clicked redirect to the appropriate section of the page, just like clicking on About and Contact takes us to the About and Contact section respectively, let's also make sure that when we click on the portfolio it takes us to the portfolio section of our page which is the My Works section :+1:

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @10-menachi @coker2019 @iamsjunaid, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
Linter checks :heavy_check_mark: 
Readme :heavy_check_mark: 
github flow :heavy_check_mark: 
Pair programming :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @10-menachi , 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**